pylxd/profile.py

```diff
@@ -44,11 +44,12 @@ def all(cls, client):
         return profiles
 
     @classmethod
-    def create(cls, client, name, config):
+    def create(cls, client, name, config, devices={}):
```

Comment:
I'd prefer it `devices` was `None` by default, and then `if config is not None:` for the logic for adding devices.